libs/sdk-backend-spi/src/workspace/execution/index.ts

```diff
@@ -293,6 +322,18 @@ export interface IExecutionResult {
      */
     readWindow(offset: number[], size: number[]): Promise<IDataView>;
 
+    /**
+     * Reads forecast for the execution result.
+     * @beta
+     */
+    readForecastAll(config: IForecastConfig): Promise<IForecastResult>;
+
+    // TODO: add support for forecast paging
```

Comment:
Probably lets remove this, implement it, or create a ticket for it.

Reply:
I will remove for now, hope will not be neccesary for now